src/Services/mediawiki.php

```diff
@@ -56,16 +56,35 @@
 	'WikiImporter' => function ( $containerBuilder, \ImportSource $importSource ) {
```

Comment:
In https://github.com/wikimedia/mediawiki/commit/64001f0ecd6ffdd8a639e4aed0289afc6bd1e539#diff-81ab53e2d79bfc77e91773c25ac399efeaa16486044dea310100d719b14b2cf9R26, it added a 2nd param called $performer. I'm not sure if we should do that here but like it's mw 1.42+ thing.

Reply:
Would `$services->getWikiImporterFactory->getWikiImporter( $importSource, RequestContext::getMain()->getAuthority() );` works across versions?

Reply:
Oh! Didn't realise we could do just that, yeh that should work. In MW 1.39 it doesn't have the 2nd param but I think it gets ignored?

Reply:
Thanks, done.